ingress_export.js

```diff
@@ -11,6 +11,8 @@
 // @include http://*ingress.com/intel*
 // @match https://*ingress.com/intel*
 // @match http://*ingress.com/intel*
+// @match https://inter.ingress.com/*
```

Comment:
Is this not covered already by the *ingress.com/intel* rules?

Reply:
No, cause now you can write de url without de */intel* part, and when that happens de plugin doesn't ... ¿function?

Reply:
And it must be "intel" ... no "inter"